### PR TITLE
Addresses #134 ("SDP-US document may not adhere to HRM")

### LINKED ISSUE #134: SDP-US document may not adhere to HRM
The statement in the abstract:

> The text profile is a superset of [ttml10-sdp-us].

is in fact untrue since it is possible to construct SDP-US documents that fail HRM requirements.

Suggest rewording to:

> The text profile is a syntactic superset of [ttml10-sdp-us]; it is feasible to create documents that are simultaneously conformant both to SDP-US and the IMSC Text profile.

or something along those lines.

### PR DESCRIPTION
**Pull request by palemieux**
Closes #134

### CODE REVIEW

**Comment by nigelmegitt** [comment]
Looks good to me.

**Comment by palemieux** [comment]
See #128 as well
